src/requirements/data/colleges/as.ts

```diff
@@ -10,7 +10,10 @@ const casRequirements: readonly CollegeOrMajorRequirement[] = [
       'they can also take all their credits in Arts & Sciences and accumulate more than 120. ' +
       'Note: AP, IB, and A-Level credits count toward the 120 total credits but not toward the 100 A&S credits.',
     source: 'https://as.cornell.edu/degree-requirements',
-    checker: (course: Course): boolean => course.catalogDistr?.includes('-AS') ?? false,
+    checker: (course: Course): boolean =>
+      course.acadGroup.includes('AS') ||
+      course.subject === 'CS' ||
+      (course.catalogDistr?.includes('-AS') ?? false),
```

Comment:
It was originally `course.acadGroup.includes('AS')`, but it was changed to the current code in #115. I don't think we should remove the `acadGroup` check. I also special cased for CS, since CS is both in arts and engineering. Not sure whether it's 100% correct...